src/pug/includes/head.pug

```diff
@@ -7,6 +7,7 @@ meta(http-equiv="X-UA-Compatible" content="IE=edge")
 
 meta(name="description" content="swag opportunities for developers")
 
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
```

Comment:
This won't work in pug, you need to adapt the syntax.
Also, I think we'd better use an image or SVG stored as data url (base64ed) in case instead of importing the whole fontawesome css bundle!